### ISSUE TITLE
Configure sticky sessions

### ISSUE THREAD

**Issue by ykostsik** [emissary-ingress/emissary#5132]
Hi all,
As per docs, I'm trying to enable sticky sessions in ambassador. 
Before we used KubernetesServiceResolver as a default one and the ambassador module
```
apiVersion: v1
items:
- apiVersion: getambassador.io/v2
  kind: Module
  metadata:
    annotations:
      meta.helm.sh/release-name: ambassador
      meta.helm.sh/release-namespace: ambassador
    labels:
      app.kubernetes.io/managed-by: Helm
      helm.sh/chart: emissary-ingress-8.4.1
    name: ambassador
    namespace: ambassador
  spec:
    config:
      add_linkerd_headers: true
      envoy_log_type: text
      preserve_external_request_id: false
kind: List
metadata:
  resourceVersion: ""
```

So I created KubernetesEndpointResolver as described in the documentation:
```
apiVersion: getambassador.io/v3alpha1
generation: 1
kind: KubernetesEndpointResolver
metadata_labels:
  ambassador_crd: dev-ep-resolver.ambassador
name: dev-ep-resolver
namespace: ambassador
```
and ambassador Mapping, that points to this EndpointResolver
```
apiVersion: getambassador.io/v2
kind: Mapping
metadata:
  annotations:
    <..>
spec:
  connect_timeout_ms: 10000
  host: mypage.example.com
  load_balancer:
    cookie:
      name: sticky-cookie
      ttl: 300s
    policy: ring_hash
  prefix: /
  resolver: dev-ep-resolver
  service: my-service.app:8080
  timeout_ms: 10000
  use_websocket: true
```

**Expected behavior**
I expected it's enough to configure sticky sessions, but it seems it does not work properly.
My assumption the Endpoint resolver should be aware of my-service.app, but I'm not sure if we need to define the EndpointResolver namespace in the Mapping.
As well as configuration precedence, I suppose that Mapping overrides the ambassador module rules, but I could be wrong there.

**Versions (please complete the following information):**
 - Ambassador: [e.g. 3.4.1]
 - Kubernetes environment [AWK EKS 1.26]

**Comment by ykostsik** [comment]
Could someone help here please?

**Comment by cindymullins-dw** [comment]
Hi @ykostsik , your assumptions are correct: the Mapping overrides the Module and generally you shouldn't need to specify the endpoint resolver namespace in the Mapping.  The config looks ok.  How is it failing - does traffic go through but is not sticky?  Or are you getting response errors?  One thing users have noted is that you have to wait for ambassador to be up & running (readiness state) before applying the Resolver. If you install the resolver immediately after or during ambassador startup it can fail.

**Comment by cindymullins-dw** [comment]
Ok, let's try this.  In the Mapping can you include the namespace under spec.service but remove the port?  And please try changing the name of the EndpointResolver to 'endpoint', and then updating the Mapping accordingly?

**Comment by ykostsik** [comment]
Hi @cindymullins-dw ,
Thanks for reply.
So, at least I was able to see active routes and clusters for my service on Ambassador Route Table diagnostic page after after I renamed the resolver as you suggested above. Seems, It still has some issues, bc I lost stickiness for minutes but it could be also the configuration issue and I'm dealing with it.
Regarding the question above, yes it reaches the pod but the app is not working without stickiness, so it returns the error page.
